### ISSUE TITLE
image type

### ISSUE THREAD

**Issue by minouei-kl** [ibm-aur-nlp/PubTabNet#17]
As mentioned in the paper, the results reported in 3 categories: simple, complex, and all.
and also in the [sample_gt.json](https://github.com/ibm-aur-nlp/PubTabNet/blob/7d9fbe9f63985d9d62adc7f396397fba9c81eef2/src/sample_gt.json#L1) 
there is a "type" property to show whether is simple or complex.
but this property is missing in the training and validation set. is there a way to distinguish between the two in the training set?

**Comment by ajjimeno** [comment]
We did not generate such information for the training and validation
sets. The criteria to label tables as simple or complex is based on tables
without or with multi-rows/columns (https://arxiv.org/pdf/1911.10683.pdf),
so it should be possible to generate it in the training and validations
sets.
